testeng/jobs/toggleSpigot.groovy

```diff
@@ -109,9 +110,9 @@ secretMap.each { jobConfigs ->
             mailer(jobConfig['email'])
             hipChat {
                 token(jobConfig['hipchat'])
-                rooms('Tools Super Team')
+                rooms('Test Engineering')
```

Comment:
I think that you will need to create a new hipchat token. If I recall correctly, the token it is using is only for 'tools super team'. Not sure